zstash/extract.py

```diff
@@ -393,6 +393,24 @@ def extractFiles(  # noqa: C901
         # True if we should actually extract the file from the tar
         extract_this_file: bool = keep_files and should_extract_file(files_row)
 
+        # TODO: implement this proof-of-concept
```

Comment:
Maybe we can implement this as a new feature later on. We could modify `zstash ls` to list the md5 values too. Maybe add a `--tars` option to that command.